<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateOrderPaper</class>
 <widget class="QDialog" name="CreateOrderPaper">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>644</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Criar Comanda | Sistema de Controle de Comandas</string>
  </property>
  <widget class="QGroupBox" name="groupBox_orderPaperCreate">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>581</width>
     <height>441</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="pushButton_orderPaperCreate">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>370</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Criar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_orderPaperCreateCancel">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>370</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancelar</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>561</width>
      <height>331</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_orderPaperCreate">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_orderPaperClientName">
       <item>
        <widget class="QLabel" name="label_orderPaperClientName">
         <property name="text">
          <string>(*) Nome do Cliente:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_orderPaperClientName">
         <property name="clearButtonEnabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_orderPaperClientAge">
       <item>
        <widget class="QLabel" name="label_orderPaperClientAge">
         <property name="text">
          <string>Idade:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_orderPaperClientAge"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_orderPaperClientTimestamp">
       <item>
        <widget class="QLabel" name="label_orderPaperTimestamp">
         <property name="text">
          <string>(*) Data e hora:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateTimeEdit" name="dateTimeEdit_orderPaperTimestamp">
         <property name="dateTime">
          <datetime>
           <hour>0</hour>
           <minute>0</minute>
           <second>0</second>
           <year>2000</year>
           <month>1</month>
           <day>1</day>
          </datetime>
         </property>
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_orderPaperItems">
       <item>
        <widget class="QLabel" name="label_orderPaperItems">
         <property name="text">
          <string>(*) Itens Pedidos:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_orderPaperItems">
         <item>
          <widget class="QPlainTextEdit" name="plainTextEdit_orderPaperItems"/>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_orderPaperNewItems">
           <property name="font">
            <font>
             <strikeout>true</strikeout>
            </font>
           </property>
           <property name="text">
            <string>Adicionar Item</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_orderPaperAdditionalValue">
       <item>
        <widget class="QLabel" name="label_orderPaperAdditionalValue">
         <property name="text">
          <string>Adicional:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_orderPaperAdditionalValue"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_orderPaperDiscountValue">
       <item>
        <widget class="QLabel" name="label_orderPaperDiscountValue">
         <property name="text">
          <string>Desconto:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_orderPaperDiscountValue"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_TotalValue">
       <item>
        <widget class="QLabel" name="label_orderPaperTotalValue">
         <property name="text">
          <string>(*) Valor total:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_orderPaperTotalValue"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="label_orderPaperCreateTitle">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>20</y>
     <width>601</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Criar Comanda</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
